app/models/thredded/messageboard.rb

```diff
@@ -3,11 +3,13 @@ class Messageboard < ActiveRecord::Base
     FILTERS = %w(markdown bbcode)
 
     extend FriendlyId
-    friendly_id :name, use: :slugged
+    friendly_id :slug_candidates,
+                use:            [:slugged, :reserved],
+                # Avoid route conflicts
+                reserved_words: %w(messageboards private-topics autocomplete-users theme-preview)
```

Comment:
Acknowledged, but can't pass a symbol here, and any method called here would need to be defined beforehand (above). We could add a constant but then it can't be private.